### PR TITLE
Refactor folder structure

### PR DESCRIPTION
**Pull request by HaGuesto**
This refactoring has two reasons:

1. We adapt the structure of a odoo community module  like in https://github.com/OCA/web
2. We save two steps in setting up the developer environment. In the future the doodba will automatically download all required repos for you into the right folder.

### CODE REVIEW

**Comment by jamescrowley** [comment]
@haGuesto can you remove the commits that have already been merged so the changes in this PR are clearer? Think you might need to rebase your local branch onto the upstream master.

**Comment by jamescrowley** [comment]
Am I right in understanding this brings the pampiraiki specific stuff back into this repo? So for a new customer we'll add to this repo too? (No specific problem with that just checking I understand!)

File: README.md [hunk 1]
```diff
@@ -24,40 +24,46 @@ Please also check out our [Code of Conduct](CODE_OF_CONDUCT.md)!
 
 1.  Install odoo v11. We created [a Boxwise dev-setup](https://github.com/boxwise/boxwise-doodba). It is based on a docker environment for development and production of odoo by Tecnativa. It should just be 5 commands and you are ready to go. You have to have [docker](https://docs.docker.com/install/) and docker-compose installed for this dev-setup.
 
-2.  Clone this folder into the [private src folder](https://github.com/boxwise/boxwise-doodba/tree/11.0_wms/odoo/custom/src/privates) of your setup. **Please do NOT rename the boxwise_wms repository!**
```
Comment (jamescrowley) [inline README.md]: Yay to less steps!

File: README.md [hunk 2]
```diff
@@ -24,40 +24,46 @@ Please also check out our [Code of Conduct](CODE_OF_CONDUCT.md)!
...
 
-4. Activate the developer mode! There two ways:
+3. Activate the developer mode! There two ways:
 
     1. Add `debug` in the url --> `localhost:11069/web?debug#....` or
-    2. Go to settings and click the link hidden beneath the credentials on the far right! 
+    2. Go to settings and click the link hidden beneath the credentials on the far right!
 
-5.  Install the `boxwise_wms` module!
+4.  Install the `boxwise_wms` module!
```
Comment (jamescrowley) [inline README.md]: Can doodba do this automatically for us too??

File: README.md [hunk 3]
```diff
@@ -24,40 +24,46 @@ Please also check out our [Code of Conduct](CODE_OF_CONDUCT.md)!
...
-    
-The following steps are required if [Issue #13](https://github.com/boxwise/boxwise_wms/issues/13) and [Issue #15](https://github.com/boxwise/boxwise_wms/issues/15) are not yet solved.
+
+The following steps are required if [Issue #15](https://github.com/boxwise/boxwise_wms/issues/15) are not yet solved.
 
 6.  Check out the links above and enable the settings / user rights by hand.
 
+## Where to go from here?
+
+If you want to check out the boxwise frontend - the fancy designs and form - go to
```
Comment (jamescrowley) [inline README.md]: 👍👍

File: README.md (hunk 2 above)
Comment (HaGuesto) [reply README.md]: I'll check that out. I imagine it might be possible.

**Comment by HaGuesto** [comment]
> Am I right in understanding this brings the pampiraiki specific stuff back into this repo, hence the additional commits? So for a new customer we'll add to this repo too? (No specific problem with that just checking I understand!)

I am not super happy with the naming of the pampiraiki repo. It is not really best practice. I rather see it as our standardization of products and categories which should evolve into a the default products which are loaded for new users. Maybe we can offer different data sets. One for clothes, one for hygiene items, ...

Should I remove all the commits until the merge @jamescrowley?

**Comment by DurkoMatko** [comment]
Works fine for me, I was able to install both submodules, open database manager, create new DB etc etc. And cherry on top - tests are now (finally) discovered when odoo is started --test-enable as well :)

**Comment by HaGuesto** [comment]
@jamescrowley I managed to kill another three steps. Everything should work now with about 7 commands from the doodba.
https://github.com/boxwise/boxwise-doodba/tree/refactor_boxwise_folder_structure

**Pull request merged**